model.py

```diff
@@ -0,0 +1,55 @@
+# получаем и отправляем все данные из текстового файла
+def get_data():
+    with open('contacts.txt', 'r', encoding='utf-8') as file:
+       data = file.readlines()
+    return data
+
+
+# добавляем контакт
+def add_contact(contact):
+    add_data = contact.split()
+    res_add_data = add_data[0] + ':' + add_data[1] + ':' + add_data[2] + ':' + add_data[3] 
```

Comment:
Вы не предусмотрели вариант, когда пользователь может ввести не то количество информации, которое вы ожидаете
![image](https://github.com/Maximus556/GB_8_telephone_directory/assets/49741194/38e229e4-9306-49a0-8640-6b391545bd26)

Reply:
Проблема решается так
```suggestion
    res_add_data = ':'.join(add_data) 
```